util/shop/items/powerups.js

```diff
@@ -2,11 +2,13 @@ export const powerups = {
   crowncard: {
     price: 110,
     description: 'Convert coins back into a crown at a premium.',
+    quantified: true,
     enabled: true,
   },
   sneak: {
     price: 60,
     description: 'Sneakily move towards the target number.',
+    quantified: false,
```

Comment:
I was able to omit `quantified: false` for the powerups that it will never make sense for and it still works as expected.
I left `quantified: false` for the ones that I can envision a scenario where that may be added in the future.

Reply:
This is epic.